baker

```diff
@@ -291,7 +298,9 @@ bake)
 	time for post in "${posts[@]}"; do
 		id="$(basename "$post" .md)"
 		# skip drafts
-		[[ "$(header draft < "$post")" == false ]] || continue
+		if [ "$(header draft < "$post")" != false ]; then
```

Comment:
Won't it be slower ?
If i rewrote with the `if...else...fi`, the shell will have to jump until the enf of `fi` and then start a new turn in the `for`.
By using the `continue`, the shell directly jmp to `for` statement.